otter/convergence/service.py

```diff
@@ -299,6 +321,7 @@ def got_children_with_stats(children_with_stats):
     return eff.on(got_children_with_stats)
 
 
+@do
```

Comment:
Man, seeing this in Python feels real weird. (That is not something you should address in this PR; I'm just saying that `@do` doesn't really help me when I first read this code even though I knew about both Haskell's `do` and `effect.do.do`.)